src/main/kotlin/baseball/controller/GameController.kt

```diff
@@ -0,0 +1,37 @@
+package baseball.controller
+
+import baseball.domain.Computer
+import baseball.domain.GameResult
+import baseball.service.GameService
+import baseball.view.InputView
+import baseball.view.OutputView
+
+class GameController(
+        private val inputView: InputView,
+        private val outputView: OutputView,
+        private val gameService: GameService
+) {
+
+    fun run() {
+        outputView.printStart()
+
+        var gameCommand: GameCommand = GameCommand.play()
+
+        while (gameCommand.isPlaying()) {
+            play(Computer.withRandomBall())
+            gameCommand = inputView.readGameCommand()
+        }
+    }
+
+    private fun play(computer: Computer) {
+        var gameResult: GameResult
+
+        do {
+            val playerBall = inputView.readPlayerBall()
+            gameResult = gameService.play(computer, playerBall)
+            outputView.printResult(gameResult)
+        } while (!gameResult.isEnd())
```

Comment:
null인 gameResult를 선언하지 않아도 돼요!
do 블럭 내에서 사용할 수 있습니다!
```suggestion
        do {
            val playerBall = inputView.readPlayerBall()
            val gameResult = gameService.play(computer, playerBall)
            outputView.printResult(gameResult)
        } while (!gameResult.isEnd())
```

Reply:
Java의 추억... 
저도 do while문 썼는데 do안에 있는 거 바로 참조가능해서 깜짝 놀랐슴미다

Reply:
와우 놀랍네요 이건 진짜 좋다요